node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
We shouldn't add a function.  If anything we should move to the model of a function call only needed in the exceptional case.  In the common and virtually all cases that I know of, a task.json or module.json is in the current dir.

Is that feasible?